### PR TITLE
[app_dart] Approve release PRs

### PR DESCRIPTION
**Pull request by CaseyHillers**
Fixes https://github.com/flutter/flutter/issues/101397

### CODE REVIEW

**Review by XilaiZhang — APPROVED** [review]
images (1)

File: app_dart/lib/src/request_handlers/github_webhook.dart [hunk 1]
```diff
@@ -181,6 +183,29 @@ class GithubWebhook extends RequestHandler<Body> {
     await scheduler.triggerPresubmitTargets(pullRequest: pr);
   }
 
+  /// Release tooling generates cherrypick pull requests that should be granted an approval.
+  Future<void> _tryReleaseApproval(
```
Comment (godofredoc) [inline app_dart/lib/src/request_handlers/github_webhook.dart:187]: Part of the release process requires that two people get involved in release PRs this change will allow unilateral changes. @Hixie thoughts?

Comment (Hixie) [reply app_dart/lib/src/request_handlers/github_webhook.dart:187]: i don't mind this for the cherrypick step, so long as the actual release (once the cherrypicks are known) requires two people to sign off.

Comment (CaseyHillers) [reply app_dart/lib/src/request_handlers/github_webhook.dart:187]: I'm less concerned about getting reviews on beta and stable releases as those are reviewed weekly. This is to unblock the g3 roller where we're CPing changes into the internal release branches.

My long term plan here is there would be no human accounts that would get sign off, but instead automated bot accounts creating CP PRs would get approved.

**Review by godofredoc — APPROVED** [review]

File: app_dart/test/src/datastore/fake_config.dart [hunk 2]
```diff
@@ -258,4 +258,7 @@ class FakeConfig implements Config {
 
   @override
   String get defaultRecipeBundleRef => 'refs/heads/main';
+
+  @override
+  Future<List<String>> get releaseAccounts async => <String>['dash-releaser'];
```
Comment (godofredoc) [inline app_dart/test/src/datastore/fake_config.dart:263]: s/dash/dart-flutter?

**Pull request merged**